src/components/Header/NavBar/NavBar.js

```diff
@@ -6,24 +6,48 @@ import ConnectedAuthorization from './Authorization/Authorization';
 export default class NavBar extends Component {
   render() {
     return (
-      <nav className="NavBar">
-        <ul>
-          <li>
-            <NavLink to="/" exact>Home</NavLink>
-          </li>
-          <li>
-            <NavLink to="/cards">Cards</NavLink>
-          </li>
-          <li>
-            <NavLink to="/products">Products</NavLink>
-          </li>
-          <li>
-            <ConnectedAuthorization
-              withAuth={logOut => <NavLink to="/login" onClick={logOut}>Log out</NavLink>}
-              withoutAuth={<NavLink to="/login">Login</NavLink>}
-            />
-          </li>
-        </ul>
+      <nav>
+        <div className="NavBar">
+          <ul className="NavBar__ul">
```

Comment:
I personally don't like class names that start with capital letter, `navbar` or `nav-bar` fits better here IMO.
There is no reason to add element name to its class name. I would go with `NavBar__list` and `Navbar__item`, there is also no need to add such deep nesting with the name.

Reply:
Thank you very much for the reviews

Reply:
How deep nesting should be, two or three?

Reply:
I would say one level of "elements" in the "block".
In css you can select items by element, no need for tons of classes, for example `.navbar__list a`

Reply:
ok, thx.